_posts/jared_baker/2018-05-04-Lets-Encrypt.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Let's Encrypt the Collaboratory"
+breadcrumb: true
+author: jared_baker
+date: 2018-05-04
+categories: jared_baker
+tags:
+    - Lets Encrypt
+    - ssl
+    - linux
+    - acme
+    - tutorial
+teaser:
+    info: SSL all your sites with Let's Encrypt and Ansible
+    image: jared_baker/letsencrypt/le-logo-standard.png
+header:
+    version: small
+    title: Software Engineering Blog
+    image: jared_baker/le-logo-standard.png
+    icon: icon-blog
+---
+## Introduction
+OICR runs the Cancer Genome Collaboratory, a cloud environment designed for cancer research and composed of open source software. The Collaboratory has many web applications that need to be secured with SSL and has done so traditionally with long life paid certificates from various trusted Certificate Authorities (Digicert, etc). We would purchase certificates from a trusted CA ($$$) then use a combination of Ansible and scp to distribute the certificates every couple years to the various systems that need them.
```

Comment:
I would replace "trusted" by "traditional", just not to give the impression that let's encrypt is less "trusted" than the traditional ones. They are disrupting the traditional model, not necessarily introducing risks.